packages/bootstrap-4/src/RadioWidget/RadioWidget.tsx

```diff
@@ -22,45 +19,43 @@ const RadioWidget = ({
 }: WidgetProps) => {
   const { enumOptions, enumDisabled } = options;
 
-  const _onChange = ({}, value: any) =>
+  const _onChange = ({
+    target: { value },
+  }: React.ChangeEvent<HTMLInputElement>) =>
     onChange(schema.type == "boolean" ? value !== "false" : value);
   const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
     onBlur(id, value);
   const _onFocus = ({
     target: { value },
   }: React.FocusEvent<HTMLInputElement>) => onFocus(id, value);
 
-  const row = options ? options.inline : false;
+  const inline = options.inline !== null ? options.inline : false;
```

Comment:
This will crash with `TypeError: Cannot read property 'inline' of undefined` if `options` prop is not passed, whereas the original implementation results in `false`.

The original implementation is quite defensive, since while` options.inline` should be a boolean, it can be any value. Later, on line 43, they further typescript cast it to boolean to get it to conform to the type interface. However, in runtime it could still be any truthy or falsy value.

Personally, I'd implement this as 
```suggestion
  const inline = Boolean(options && options.inline);
```
Which ensures that the value is coerced to boolean both at typecheck time and runtime. This should allow us to remove the `as` cast on like 43, too.

Reply:
@jevakallio Thanks for the suggestion :)